README.md

```diff
@@ -1,63 +1,67 @@
-# siddhi-map-text
+siddhi-map-text
 ======================================
+
+The **siddhi-map-text extension** is an extension to <a target="_blank" href="https://wso2.github
+.io/siddhi">Siddhi</a> that provides the functionality to convert canonical events of the server in the WSO2Event format to any text message format and convert events of any text format to the server's canonical event format (WSO2Event) for processing.
+
+Find some useful links below:
+
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-text">Source code</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-text/releases">Releases</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-text/issues">Issue tracker</a>
+
+## Latest API Docs 
+
```

Comment:
Latest API docs need to come here. Refer unique docs

Reply:
Fix in d6dc7df